packs/excel/config.schema.yaml

```diff
@@ -0,0 +1,30 @@
+---
+  excel_file: 
```

Comment:
Thanks.

Let me know when you have pushed all the changes and I will have a look.